### ISSUE TITLE
Make the step buttons into tabs

### ISSUE THREAD

**Issue by brodecva** [odalic/odalic-ui#418]
They are not steps actually because the order is not that important. So it might be worthwhile to change the into a tab style with tabs "Table", "Subject columns", "Relations".

**Comment by istvan-satmari** [comment]
WONTFIX

I tried that, you can see branch "taskresult-tabset" with the work in progress.

So, why won't it be finished?
Changing the "div-switch" directives int "tabset" introduced a lot of bugs. A certain bug arises from a weird order of subcomponent loading, but that happens only ocassionally (makes the "export" buttons stop working). The more troublesome bug is connected with graph component (possibly mainly with the D3 library itself):
on Firefox I get NS_ERROR_FAILURE exception (makes the browser completely freeze in seconds)
on Chrome the graph gets slightly distorted, the lock icons are moved further away from the edges and label on nodes are moved from the center as well
The actual cause of the bugs is rather hard to find out, but it probably has to do with the way the DOM is built now when the tabset directive is introduced.
As for the leaving the basic buttons and just moving them to the top of the page, I stumbled upon another kind of bug.
So basically my decision no to fix these issues comes down to "don't fix what's not broken". Especially a week before the deadline.

**Comment by brodecva** [comment]
OK.

**Comment by istvan-satmari** [comment]
nvm, it seems I fixed the issue.
To elaborate:
The problem with UI tabset is that it only hides the portion of created DOM instead of actually removing it, unlike built-in angular ng-switch. Some of our components (e.g. the graph) relied on this behaviour, which led to problems.
I basically mixed ng-switch (actually, I used ng-if, though the behaviour is the same) with the tabset component and now it seems to be working fine.
